package.json

```diff
@@ -220,6 +220,11 @@
               "trace.server": "off"
             }
           },
+          "terraform.languageServer.ignoreSingleFileWarning": {
+            "scope": "resource",
+            "type": "boolean",
+            "description": "Path to the Terraform binary"
+          },
```

Comment:
Is there any benefit to setting `default: false` here as well?

Reply:
Good thing you asked, I missed adding a description here.

As for the setting, the `const ignoreSingleFileWarning = config('terraform').get<boolean>('languageServer.ignoreSingleFileWarning', false);` line in `clientHandler.ts` will ensure we get false if nothing is specified, but it's a good idea to be explicit in both places so I added it.